src/style.css

```
@@ -0,0 +1,40 @@
+.body {
+  margin: 0;
+  box-sizing: border-box;
+  font-size: 0;
+}
+
+.container {
+  display: flex;
+  flex-flow: column wrap;
+}
+
+.star__star {
+  display: inline-block;
+  margin: 2px 6px 2px 2px;
+  height: 12px;
+  width: 12px;
+
+  background-image: url(./images/star.svg);
+  background-size: cover;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
```

you are using the same styles for these classes, better group them and make the code cleaner

```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n+2),
.stars--2 :nth-child(-n+3),
.stars--2 :nth-child(-n+4),
.stars--2 :nth-child(-n+5) {
  background-image: url(./images/star-active.svg);
}
```